app/src/main/java/app/gamenative/ui/PluviaMain.kt

```diff
@@ -111,8 +174,9 @@ fun PluviaMain(
                 Timber.i("[PluviaMain]: Processing pending launch request for app ${launchRequest.appId} (user is now logged in)")
 
                 // Check if the game is installed
-                if (!SteamService.isAppInstalled(launchRequest.appId)) {
-                    val appName = SteamService.getAppInfoOf(launchRequest.appId)?.name ?: "App ${launchRequest.appId}"
+                val gameId = createLibraryItemFromAppId(launchRequest.appId).gameId
+                if (!SteamService.isAppInstalled(gameId)) {
```

Comment:
seems like overkill to create a library item here. better to use the container method to get the gameId from the appId (if I remember correctly there is one, otherwise I'll make one)